maintainer-guide.md

```diff
@@ -0,0 +1,79 @@
+  2. Describe the solution in detail with links to the original issue and any related issues that it might fix or close. 
+  3. GitHub Draft PRs are a great way to get CI or human feedback on work that isn't yet ready to merge. PRs can be created as drafts and converted to normal PRs once the CI passes. More information about GitHub Draft PRs: https://github.blog/2019-02-14-introducing-draft-pull-requests/
+  
+
+- PRs should be merged once they
+  1.  **pass the automated tests** (GitHub Actions, CLA signing, etc.),
+  2.  have the **review comments addressed**,
+  3.  get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
+  4.  have been open for at least **24 hours** unless the changes are trivial
+
+- To merge a pull request, it must have at least:
+
+  - one approval for simple documentation fixes
+  - two approvals for everything else
+
+- When merging a PR, you should:
+
+  1. Use the **merge strategy that produces a clean Git history**: ["Squash and merge"](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) commits and ensure the resulting commit message is:
+    - descriptive
+    - sentence case
+    - If instead the PR author took the time to craft individual, informative messages for each commit, then use the `Rebase and merge` method,to honor that work and preserve the history of the changes.
+    - For less clear-cut cases, a simple heuristic you can follow is that if there are more "dirty" commits than "clean" commits,then prefer squash, else do a rebase.
+
+  2. Update the [changelog](https://github.com/hirosystems/stacks-blockchain-api/blob/master/CHANGELOG.md) directly via the [GitHub website](https://github.com/stylelint/stylelint/edit/master/CHANGELOG.mdhttps://github.com/hirosystems/stacks-blockchain-api/edit/master/CHANGELOG.md)
+    1. Group the item under and in the following order: "Features", "Bug Fixes", "Reverts", "Chores", "Documentation", 
+    2. All Breaking Changes **must** be listed at the top of the changelog under "⚠ BREAKING CHANGES", followed by any "Compatibility Issues" or "Known Limitations". This section serves as a notification that this isn't a straightforward upgrade.
+    3. Order the item within the group by the most-significant first to the smallest, and then alphabetically by item name.
+    4. Suffix the item with the relevant issue number and pull request number, 
+  3. Post this update as a comment to the pull request.
+  
+
+- If a PR fails to get a review from a second maintainer after a few days, the first maintainer should ping others for review. If it still lingers around for **over a week without a second maintainer’s approval**,the first maintainer can go ahead and merge it.
+
+- If the only issues holding up a merge are **trivial fixes**
+  (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
```

Comment:
I'm good with removing "and the author doesn't respond in a day or two" and having this happen immediately.